tg/utils.py

```diff
@@ -126,16 +126,12 @@ def num(value: str, default: Optional[int] = None) -> Optional[int]:
         return default
 
 
-def is_yes(resp: Optional[str]) -> bool:
-    if not resp or resp.strip().lower() == "y":
-        return True
-    return False
+def is_yes(resp: str) -> bool:
+    return not resp or resp.strip().lower() == "y"
```

Comment:
having `not resp` does not make sense for ` resp: str`, `return resp.strip().lower() == "y" ` should be enough

Reply:
Not really. We have default answer [Y/n] which returns empty string when enter pressed

Reply:
I see, ok!